notify.js

```diff
@@ -45,7 +45,9 @@
             notifyClose: null,
             notifyClick: null,
             notifyError: null,
-            timeout: null
+            timeout: null,
+            requireInteraction: false,
+            closeOnClick: false // used with the requireInteraction to close the notification after it's been clicked
```

Comment:
Testing shows that Chrome does not close notifications on click by default, irrespective of `requireInteraction`. I'm fine to leave `closeOnClick` as an option, but the comment here is not strictly accurate. I think we can remove it.